iasylum/bot-code.scm

```diff
@@ -155,14 +157,15 @@
   ;;(define inq (make-queue))(define outq (make-queue))(define mybot (irc/work-queue-bot 'in-work-queue: inq 'out-work-queue: outq 'name: "igorhvr" 'server-hostname: "localhost-ia" 'server-port: 16667 'server-password: "igorhvr/ia:igorhvr" 'channel: "ia"))
 
 (define* (bot-on-channel-command-processor
-          (inq: inq) (outq: outq) (command-prefix-list: command-prefix-list))
+          (inq: inq) (outq: outq) (command-prefix-list: command-prefix-list)
```

Comment:
:nail_care: To keep styling consistent with `slack/create-bot-on-channel` line-break after each parameter.
```suggestion
          (inq: inq)
          (outq: outq)
          (command-prefix-list: command-prefix-list)
```